docroot/modules/custom/mass_content/src/Commands/MassContentCommands.php

```diff
@@ -126,4 +126,95 @@ public function migrateDateFields(string $type = '', int $limit = 0) {
 
   }
 
+  /**
+   * Migrate Service data.
+   *
+   * @param int $limit
+   *   Number of nodes to process
+   *   Argument provided to the drush command.
+   *
+   * @command mass-content:migrate-service
+   *
+   * @usage mass-content:migrate-service 1000
+   *   1000 is the number of nodes that will be processed.
+   */
+  public function migrateServiceData(int $limit = 0) {
+    $fields = [
+      'field_service_ref_actions_2',
+      'field_link_group',
+      'field_service_ref_actions',
+      'field_service_key_info_links_6',
+      'field_template',
+      'field_service_ref_locations'
+    ];
+    // 1. Log the start of the script.
+    $this->loggerChannelFactory->get('mass_content')->info('Update nodes batch operations start');
```

Comment:
The Drush way is `$this->logger()->info()`

Reply:
Fixed, thank you